pkgs/by-name/go/gokey/package.nix

```diff
@@ -1,16 +1,18 @@
 { lib, buildGoModule, fetchFromGitHub }:
 buildGoModule {
   pname = "gokey";
-  version = "0.1.2-unstable-2023-11-16";
+  version = "0.1.3";
+
+  patches = [ ./version.patch ];
 
   src = fetchFromGitHub {
     owner = "cloudflare";
     repo = "gokey";
-    rev = "26fcef24d123e0eaf7b92224e6880f529f94aa9f";
-    hash = "sha256-nt4fO8NKYfRkpoC1z8zDrEZC7+fo6sU/ZOHCMHIAT58=";
+    rev = "b5342a3cf158fa31ce4b6087955f7625b16b2031";
```

Comment:
```suggestion
    rev = "refs/tags/${version}";
```

Reply:
The majority of packages in nixpkgs use the version tag, I suggest we just follow suit.